addons/html_builder/static/src/website_builder_action.scss

```diff
@@ -54,3 +58,7 @@
         }
     }
 }
+
+body:has(.o_builder_sidebar_open) .o_notification_manager {
+    @include o-position-absolute($top: map-get($spacers, 2), $right: calc(#{$o-we-sidebar-width} + 0.5rem));
+}
```

Comment:
space

Reply:
this is because of prettier :smiling_face_with_tear: I cannot add it back :grimacing: